text/0000-deprecate-cargo-categories.md

```diff
@@ -0,0 +1,131 @@
+- Feature Name: (fill me in with a unique ident, `my_awesome_feature`)
+- Start Date: (fill me in with today's date, YYYY-MM-DD)
+- RFC PR: [rust-lang/rfcs#0000](https://github.com/rust-lang/rfcs/pull/0000)
+- Rust Issue: [rust-lang/rust#0000](https://github.com/rust-lang/rust/issues/0000)
+
+# Summary
+[summary]: #summary
+
+Categories for crates are now deprecated and implicitly added as keywords instead. A new set of policies is added to allow the crates.io team to curate the way keywords are presented, replacing features such as the "Popular Categories" list on crates.io with a "Popular Keywords" instead.
```

Comment:
I'm wondering if we should split this RFC up into two RFCs... 🫣

The current one is conflating 1) crates.io no longer showing any categories on the webpage and ignoring them for published crates, and 2) deprecating the `categories` field in `Cargo.toml` for all registries.

The first one is more specific to crates.io, while the second has larger consequences also for third-party registries and cargo.

While there might be some overlap I'm wondering if the first part would be easier to get consensus on, and then the second part may need more discussion, also with e.g. some third party registry maintainers.

Disclaimer: I'm not saying that we absolutely have to split the RFC up, but I'm wondering what others think about that. It might also resolve the "what team is responsible for this RFC" question, with crates.io being responsible for the first part and the cargo team for the second part?

Reply:
For me, if we split it, it would be for procedural reasons but that they should be approved hand-in-hand.

I feel that approving one without ahead of the other applies an innate pressure on the other team to approve something.

Using an extreme to illustrate my point: Say the cargo team put up and approved a RFC for removing categories from manifests all by itself.  In a way that would compel the crates.io team to do *something*.